tests/html/d3ViewComponent-Tests.html

```diff
@@ -8,13 +8,14 @@
 
         <script src="../lib/infusion/lib/qunit/js/qunit.js"></script>
         <script src="../../src/lib/infusion/infusion-custom.js"></script>
-        <script src="../../src/lib/d3/d3.js"></script>
+        <script src="../../src/lib/ext/d3/d3.js"></script>
```

Comment:
Still feel it would be less to strange to have d3 and flocking libraries in ```lib/``` instead of ```lib/ext/```. What do you think?